### PR TITLE
fix: Navbar carets should never appear on a separate line

### LINKED ISSUE #369: Navbar carets should never appear on a separate line
The items in the navbar menu are not properly aligned center when transitioning from the `mobile` to the `desktop` screen. Here is the screenshot of the bug:

issue

### PR DESCRIPTION
**Pull request by MoazMirza-13**
[fixed the alignment issue.webm](https://github.com/electron/website/assets/134006296/a701541f-b8c0-4f64-9b92-1da76765d80a)

fixed the "Navbar carets should never appear on a separate line #369", now everything's working fine.  
You can also see in the video.

fixed #369

### CODE REVIEW

**Comment by MoazMirza-13** [comment]
@erickzhao @erikian @malept @Kilian @molant @VerteDinde @dsanders11 @ckerr @MarshallOfSound @BlackHole1 
can anyone give it a review.

**Comment by dsanders11** [comment]
@MoazMirza-13, please do not tag maintainers like that, it is not productive. The PR will be reviewed in due time.

**Review by BlackHole1 — APPROVED** [review]
LGTM

**Review by Kilian — CHANGES_REQUESTED** [review]
Do not remove custom variables only to replace them with hardcoded specific values. If the variables are wrong, you need to update those and make sure they work everywhere.

Changing the margins isn't a proper fix for this issue. You need to update the breakpoints, deal with whitespace and make sure the electron title remains visible.

File: src/css/custom.scss [hunk 1]
```diff
@@ -121,6 +121,9 @@
     padding: 0;
     margin: var(--ifm-navbar-item-padding-vertical)
       var(--ifm-navbar-item-padding-horizontal);
+    @media screen and (max-width: 1033px) {
+      margin: 0.25rem 0.64rem;
```
Comment (Kilian) [inline src/css/custom.scss:125]: The fix for this issue isn't hardcoding different margins, so this shouldn't be accepted.

**Review by Kilian — CHANGES_REQUESTED** [review]

File: src/css/custom.scss (hunk 1 above)
Comment (MoazMirza-13) [reply src/css/custom.scss:125]: Can you help me out ? 
I tried to find where these variables are defined but I am not able to find them

**Comment by dsanders11** [comment]
Superseded by #530.

**Pull request closed without merging**